src/test/java/app/linguistai/bmvp/service/stats/UserLoggedDateServiceTest.java

```diff
@@ -0,0 +1,171 @@
+package app.linguistai.bmvp.service.stats;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+import java.sql.Date;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import app.linguistai.bmvp.exception.NotFoundException;
```

Comment:
Eline saglık! You don't have to write tests. Since we are so close to the deadline, you can just test the endpoints via Postman. 😄